### PR TITLE
Scala/Java code fix Random with secureRandom

### PR DESCRIPTION
**Pull request by Litchilitchy**
## Description
Replace all the `Random` in Java/Scala code with `SecureRandom` (if replacable)

### CODE REVIEW

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/feature/dataset/image/BGRImgRdmCropper.scala [hunk 1]
```diff
@@ -26,7 +26,7 @@ object BGRImgRdmCropper {
 }
 
 /**
- * Random crop a specified area from the Image. The result is also an image
+ * secureRandom crop a specified area from the Image. The result is also an image
```
Comment (glorysdj) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/feature/dataset/image/BGRImgRdmCropper.scala:29]: don't need to change Random in the comments

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/utils/python/api/PythonBigDL.scala [hunk 2]
```diff
@@ -2985,7 +2985,7 @@ class PythonBigDL[T: ClassTag](implicit ev: TensorNumeric[T]) extends Serializab
                           mirror: Boolean, cropperMethod: String,
                           channels: Int)
   : RandomCropper = {
-    if (cropperMethod == "Random") {
+    if (cropperMethod == "secureRandom") {
```
Comment (glorysdj) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/utils/python/api/PythonBigDL.scala:2988]: this is also not needed to change

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/feature/transform/vision/image/augmentation/ColorJitter.scala [hunk 3]
```diff
@@ -18,6 +18,7 @@ package com.intel.analytics.bigdl.dllib.feature.transform.vision.image.augmentat
 
 import com.intel.analytics.bigdl.dllib.utils.RandomGenerator.RNG
 import com.intel.analytics.bigdl.dllib.feature.transform.vision.image.{FeatureTransformer, ImageFeature}
+
 import scala.util.Random
```
Comment (qiuxin2012) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/feature/transform/vision/image/augmentation/ColorJitter.scala:22]: Delete   this import

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**